src/CarbonSDK.ts

```diff
@@ -391,7 +420,8 @@ class CarbonSDK {
     if (!wallet.initialized) {
       try {
         // Perform initialize function as per normal, but add try-catch statement to check err message
-        await wallet.initialize(this.query);
+        const fees = await this.getGasFee()
```

Comment:
Think this line can check if this.gasFee is initialised alr, else run `await this.getGasFee`.
i.e.
```typescript
const fees = this.gasFee ?? await this.getGasFee();
```